src/LAMMPS.jl

```diff
@@ -233,6 +251,51 @@ function command(lmp::LMP, cmd::Union{String, Array{String}})
     check(lmp)
 end
 
+"""
+    create_atoms(
+        lmp::LMP, x::Matrix{Float64}, id::Vector{Int32}, types::Vector{Int32};
+        v::Union{Nothing,Matrix{Float64}}=nothing,
+        image::Union{Nothing,Vector{IMAGEINT}}=nothing,
+        bexpand::Bool=false
+    )
+
+Create atoms for a LAMMPS instance. 
+x contains the atom positions and should be a 3 by Matrix{Float64}, where n is the number of atoms. 
+id contains the id of each atom and should all be Vector{Int32} with length n.
+types contains the atomic type (LAMMPS number) of each atom and should all be a Vector{Int32} with length n.
+v contains the associated velocities and should be a 3 by Matrix{Float64}.
+image contains the image flags for each atom and should be Vector{IMAGEINT} with length n.
+bexpand is a Bool that defines whether or not the box should be expanded to fit the input atoms (default not).
```

Comment:
```suggestion
Create atoms for a LAMMPS instance. 
`x` contains the atom positions and should be a 3 by `n`,  `Matrix{Float64}`, where `n` is the number of atoms. 
id contains the id of each atom and should all be `Vector{Int32}` with length `n`.
types contains the atomic type (LAMMPS number) of each atom and should all be a `Vector{Int32}` with length `n`.
`v ` contains the associated velocities and should be a 3 by `n` Matrix{Float64}.
`image` contains the image flags for each atom and should be `Vector{IMAGEINT}` with length `n`.
`bexpand` is a `Bool` that defines whether or not the box should be expanded to fit the input atoms (default not).
```

Reply:
I modified what you have here slightly. Let me know if I should change it back.

Reply:
There is an extra space after `v`

Reply:
Ok, that last push should fix it.